README.md

```diff
@@ -1,2 +1,27 @@
 # Dim Sum - QC Tracking Dashboard
 
+## Build Requirements
+
+* Java 11 JDK
+* Maven 3.8+
+* Node 17.3+
+* NPM 8.3+
+* TSC 4.5+
+
+## Build/Run
+
+Maven is configured to automatically install Node modules via `npm`, compile TypeScript via `tsc`,
+and include requirements from both in the Java build.
+
+Build runnable `.jar` file: `mvn clean package`
+
+Run server on default port (8080): `mvn spring-boot:run`
+
+Run server on different port: `mvn spring-boot:run -Dspring-boot.run.jvmArguments="-Dserver.port=8081"`
+
+## Monitoring Endpoints
+
+The following endpoints are available on the deployed webapp for monitoring purposes.
+
+* Health status: `/actuator/prometheus`
+* Prometheus metrics: `/actuator/prometheus`
```

Comment:
Why `/prometheus` and not `/metrics` like our other systems expose?

Reply:
This is the default endpoint Spring Boot uses. It's possible to change it via properties, but I figured keep it as simple as possible. I don't think it really matters for the Prometheus-side config?

Reply:
I'm sure it's possible to change the prometheus config to go looking at this endpoint, but then humans have to remember it too - but maybe I'm the odd one out in actually searching for something on the /metrics pages on occasion?

Reply:
That's a fair point. I like having it under `/actuator/` because there are other actuator endpoints that we may choose to enable in the future, and having them all under the same path makes the security config simpler. I'll move `/actuator/prometheus` to `/metrics` as a special case though

Reply:
Prometheus metrics are at `/metrics` now. This required changing the base path for all actuator endpoints, so now we have to specify a full path for any that we enable, but that's obvious enough if/when we enable others since the config is in the same place. I disabled the health endpoint since we don't have a use for it - Prometheus will be used to monitor health